### ISSUE TITLE
Repositories for Linux distributions install

### ISSUE THREAD

**Issue by datverse** [dragonflydb/dragonfly#3000]
Hi, 
We are integrating Dragonfly into our open source software that has many users. We are facing a problem: Dragonfly currently does not have Repositories to support installation. This makes it difficult for current development as well as updating our software with your Dragonfly in the future. Can you build Dragonfly Repositories specifically to support Linux distributions?

Thanks ❤️

**Comment by romange** [comment]
We will gladly cooperate and help hosting such repos but we do not have in-house expertise in this domain.

**Comment by datverse** [comment]
Hi, @romange 
Thank you very much for your reply ❤️. We will discuss this issue with engineers. If we can help, please accept this help.

**Comment by datverse** [comment]
Hi, @romange 
I have found an immediate solution, which is to ask for help from EPEL. Let's try contacting fedoraproject.org and ask them to create a repo for Dragonfly on EPEL.
https://docs.fedoraproject.org/en-US/epel/
https://docs.fedoraproject.org/en-US/epel/epel-package-request/

**Comment by romange** [comment]
it's not gonna work because Dragonfly license is not OSI compliant. We must maintain our own repos.

**Comment by romange** [comment]
This is the package build file: 
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=dragonflydb

**Comment by datverse** [comment]
> This is the package build file: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=dragonflydb

@romange Yup, this is the package build for Arch Linux.

https://repology.org **is the packaging hub**

`Repology monitors a huge number of [package](https://en.wikipedia.org/wiki/Package_(package_management_system)) [repositories](https://en.wikipedia.org/wiki/Software_repository) and other [sources](https://repology.org/#repos-footnote) comparing packages versions across them and gathering other information. Repology shows you in which repositories a given project is packaged, which version is the latest and which needs updating, who maintains the package, and other related information. Repology might be useful`

**Comment by romange** [comment]
Yes, I've read a bit about ArchLinux - they allow non-OSI packages but this won't be the case for fedora/debian distributions, so I am afraid it won't be possible to add DragonflyDB to standard repos.

**Comment by datverse** [comment]
This issue is being clarified and gradually resolved. Can you tell me more about non-OSI packages. The links talk about "non-OSI packages".

**Comment by romange** [comment]
Please read our [FAQ](https://github.com/dragonflydb/dragonfly/blob/main/docs/faq.md#what-is-the-license-model-of-dragonfly-is-it-an-open-source)

**Comment by romange** [comment]
Yes, it's all true. And that's what I asked to help with - we can host the repo but we would appreciate the help with all the code around building such repo and pushing it to a static storage. Basically, we've never done such a thing (hosting a custom repo) and would like some help from someone who has experience with that.
